update.go

```diff
@@ -6,20 +6,24 @@ import "strings"
 type Update struct {
```

Comment:
Remember to add the handler constants:
https://github.com/tucnak/telebot/blob/9c4cc7713e1c842cd2d69ebdfc758d84cb88475b/telebot.go#L45

And update the Context:
https://github.com/tucnak/telebot/blob/9c4cc7713e1c842cd2d69ebdfc758d84cb88475b/context.go#L15